setup.py

```diff
@@ -1,17 +1,13 @@
 import sys
 from setuptools import setup, find_packages
 
-PY3 = sys.version_info[0] == 3
 
 install_requires = [
         "setuptools",
-        "robotframework-selenium2library",
```

Comment:
I'm afraid that we still need to keep this dependency, probably as

"robotframework-selenium2library>=3.0.0b1"

Probably in the future we should follow the path of Selenium2Library and create a new package with name robotframework-seleniumscreenshots